_posts/platform/app/2000-01-01-run-scheduled-tasks.md

```diff
@@ -1,28 +1,34 @@
 ---
-title: Run Scheduled Tasks
+title: Private Beta - Run Scheduled Tasks
 nav: Cron Tasks
-modified_at: 2021-09-07 00:00:00
+modified_at: 2021-09-28 17:00:00
 tags: app
 index: 15
 ---
 
 {% include upon_request.md %}
 
-Your application may need to execute some tasks at regular interval. The cron feature is here to help you run scheduled tasks based on [one-off containers]({% post_url platform/app/2000-01-01-tasks %}).
+Your application may need to execute some tasks at regular interval. The cron feature is here to help you run scheduled
+tasks based on [one-off containers]({% post_url platform/app/2000-01-01-tasks %}).
 
-Instead of running 24/7 a container to execute a task from time-to-time, the cron feature lets you start one-off containers only when you need.
+Instead of running 24/7 a container to execute a task from time-to-time, the cron feature lets you start one-off
+containers only when you need.
+
+TBD how is it enabled
```

Comment:
No idea what to write here

Reply:
I think Yann meant "how do you enable the cron product", wondering whether there are some extra step to enable the cron.json detection or not. I think you can remove it:
```suggestion
```